homeassistant/components/daikin/climate.py

```diff
@@ -266,3 +269,32 @@ def swing_list(self):
     def device_info(self):
         """Return a device description for device registry."""
         return self._api.device_info
+
+    @property
+    def is_on(self):
+        """Return true if on."""
+        return self._api.device.represent(
+            HA_ATTR_TO_DAIKIN[ATTR_OPERATION_MODE])[1] != STATE_OFF
```

Comment:
We shouldn't couple the home assistant constants with the daikin api, ie don't compare the result of the daikin api call to `STATE_OFF` which is home assistant specific.

Reply:
`STATE_OFF` translates to `off` and I thought that would be ok to reuse..

Reply:
Make a new PR and create a local constant in the daikin integration that we use for the daikin api.

The risk is that home assistant changes its constant and then this integration will break. Good practice says that we shouldn't couple the API:s like that.